build/validate-install-script-sh.yml

```diff
@@ -0,0 +1,36 @@
+parameters:
+- name: repo
+  type: string
+
+jobs:
+- job: validateLineEndings
+  steps:
+  - checkout: ${{ parameters.repo }}
+  - bash: |
+      if [grep $'\r' ./helpers/installcredprovider.sh]; then echo "CRLF line ending found" ; exit 1; fi
+    workingDirectory: $(Build.SourcesDirectory)
+    displayName: Validate Install Script
+- job: LinuxInstallDefault
+  steps:
+  - checkout: ${{ parameters.repo }}
+  - bash: | 
+      ./helpers/installcredprovider.sh -Force >> ./output.log
+      cat ./output.log
+      if [! grep "Microsoft.Net6.NuGet.CredentialProvider" ./output.log]; then echo "Expected credential provider not found"; exit 1; fi
```

Comment:
these may not be working?

https://dev.azure.com/mseng/PipelineTools/_build/results?buildId=29517904&view=logs&jobId=7bb87cd9-8286-5ea9-2f11-f451270b3282&j=53d0a6f0-ba96-53b1-d275-90a967023604&t=f0b437b2-a636-534a-c0af-e092fe23e307

Reply:
fixed